docs/antora/modules/ROOT/pages/management/source.adoc

```diff
@@ -1,120 +1,418 @@
+│ 0   │ lisk-service-gateway                    │ default     │ 0.7.0-… │ fork    │ 33326    │ 0s     │ 15   │ online    │ 0%       │ 65.6mb   │ XYZ      │ disabled │
+│ 7   │ lisk-service-market                     │ default     │ 0.7.0-… │ fork    │ 33351    │ 0s     │ 15   │ online    │ 0%       │ 36.9mb   │ XYZ      │ disabled │
+│ 6   │ lisk-service-transaction-statistics     │ default     │ 0.7.0-… │ fork    │ 33348    │ 0s     │ 15   │ online    │ 0%       │ 43.5mb   │ XYZ      │ disabled │
+└----------------------------------------------------------------------------------------------------------------------------------------------------------------------------┘
+----
+====
 
-== Reset the Lisk Service database
 
-To reset the database of Lisk Service, drop the respective MYSQL and Redis databases.
 
-=== Drop the MYSQL database
+=== Delete
+Remove all processes from the PM2 list.
 
 [source,bash]
 ----
-# Stop Lisk Service
-$ npm stop
-# Login to MYSQL with the lisk user
-$ mysql -u lisk -ppassword
-# Drop the database
-mysql> drop database lisk;
-# Create fresh database
-mysql> create database lisk;
-mysql> quit; # Quit MYSQL
+pm2 delete ecosystem.config.js
 ----
 
-NOTE: When Lisk Service is started again after a database reset, then the process to reindex all the data is initiated. This can be quite time-consuming and may take up to 2 hours.
 
-=== Flush Redis DB
+.Response
+[%collapsible]
+====
+.Example output
+[source,bash]
+----
+[PM2] [lisk-service-gateway](0) ✓
+[PM2] [lisk-service-blockchain-app-registry](1) ✓
+[PM2] [lisk-service-blockchain-indexer](3) ✓
+[PM2] [lisk-service-blockchain-connector](2) ✓
+[PM2] [lisk-service-blockchain-coordinator](4) ✓
+[PM2] [lisk-service-fee-estimator](5) ✓
+[PM2] [lisk-service-transaction-statistics](6) ✓
+[PM2] [lisk-service-market](7) ✓
+[PM2] [lisk-service-export](8) ✓
+┌----------------------------------------------------------------------------------------------------------------------------------------------┐
+│ id  │ name      │ namespace   │ version │ mode    │ pid      │ uptime │ ↺    │ status    │ cpu      │ mem      │ user     │ watching │
+└----------------------------------------------------------------------------------------------------------------------------------------------┘
+[PM2][WARN] Current process list is not synchronized with saved list. App lisk-service-gateway lisk-service-blockchain-app-registry lisk-service-blockchain-connector lisk-service-blockchain-indexer lisk-service-blockchain-coordinator lisk-service-fee-estimator lisk-service-transaction-statistics lisk-service-market lisk-service-export differs. Type 'pm2 save' to synchronize.
+----
+====
+
+== Resetting Lisk Service
 
-Reset the databases for Redis after dropping the MYSQL database:
+=== Clean all run-time files with dependencies
 
 [source,bash]
 ----
-redis-cli flushall
+make clean
```

Comment:
Since, we're discussing only about the pm2 managed processes, lets specify how to clean the locally installed dependencies with `make clean-local`. `make clean` will additionally remove the docker images, if they exist, which is unintended.
```suggestion
make clean-local
```